tools/rna_tools/cofold/cofold.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="cofold" name="Cofold" version="0.0.1">
     <description>An RNA secondary structure prediction method that takes co-transcriptional folding into account</description>
     <requirements>
-        <requirement type="package" version="0.0.1">cofold</requirement>
+        <requirement type="package" version="2.0.4">cofold</requirement>
     </requirements>
     <version_command>CoFold --version</version_command>
```

Comment:
Because CoFold has this annoying version error, you can also include something like `<version_command>echo "2.0.4"</version_command>`